### ISSUE TITLE
Feature request - more controlled way for movevapp

### ISSUE THREAD

**Issue by nekas123** [Calsoft-Pvt-Ltd/NSX_V2T_Migration_Tool#5]
Hello,
Thank you for your continued efforts on this tool. It would be great to implement a feature that provides a more controlled approach for moving vApps, particularly for tenants with a large number of VMs.

**Comment by CloudJunkieScripter** [comment]
Hi ,
yes it would be great - we are facing big customers with a lot VMs also and need to migrate workload not in one shoot. So it would be great if there would be a better controlled way to move vapps.

**Comment by nekas123** [comment]
We tried to build XML and then do API for every vAPP using this as example - https://fojta.wordpress.com/2022/03/23/how-to-move-live-vapps-across-org-vdcs/
But requires lot of effort.

**Comment by TS-hub** [comment]
I agree, more control over large footprint vDCs would be great to convince tenant owners with mission critical applications

**Comment by TheGabeMan** [comment]
@nekas123 Did you ever get this scripted? I've done several attempts but rewriting those XML files is a pain.
